src/reducers/BoardPage.js

```diff
@@ -339,14 +341,14 @@ const postprocessGetArticleList = (myId, result, isFirstFetch, usersInfo) => {
       NBoo:             each.NB,
       NPush:            each.NP,
       Title:            each.Title,
-      CreateTS:         each.CreateTS ? each.CreateTS : utils.emptyTimeStamp(),
-      UpdateTS:         each.UpdateTS ? each.UpdateTS : utils.emptyTimeStamp(),
+      CreateTS:         createTS,
+      UpdateTS:         each.UpdateTS ? each.UpdateTS : createTS,
       LastSeen:         each.L ? each.L : utils.emptyTimeStamp(),
-      CommentCreateTS:  each.c ? each.c : utils.emptyTimeStamp(),
+      CommentCreateTS:  each.c && !utils.isNullTimeStamp(each.c) ? each.c : createTS,
```

Comment:
should be `each.c && !utils.isNullTimeStamp(each.c) ? each.c : UpdateTS,` instead?

Reply:
I don't have preference, so I just changed it to `updateTS` as you thought ;)